docs/sources/auth/planning/index.md

```diff
@@ -122,6 +122,10 @@ Service account access tokens inherit permissions from the service account.
 
 ### 🚧 API keys
 
+Grafana API keys can be used to interact with data sources via HTTP APIs. API keys can have a well-defined and limited scope to resources with the help of [Roles].
+
+> **Note:** If you use Grafana v8.5 or newer, use service accounts instead of API keys. For more information, refer to [Grafana service accounts]({{< relref "../service-accounts/" >}}).
```

Comment:
oh nvm, you do point to the service account docs. :)

Reply:
I actually think that we should push this info higher (maybe even in the first sentence?) and be very explicit that API keys are deprecated in favour of service accounts.

Reply:
I moved and rephrased the warning to the top level.